README.md

```diff
@@ -1 +1,4 @@
-# project_review
\ No newline at end of file
+# project_review
+
+#Created index.html file.
+#Created styles.css file.
```

Comment:
Please, provide a more descriptive ReadMe.md file. Checkout this [template](url) for proper set up.